sphinx_rtd_theme/layout.html

```diff
@@ -207,6 +207,20 @@
   </script>
   {% endif %}
 
+  {% if theme_analytics_id %}
+  <!-- Theme Analytics -->
+  <script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', '{{ theme_analytics_id }}', 'auto');
+  ga('send', 'pageview');
+  </script>
+
+  {% endif %}
```

Comment:
This conflicts with our insertion of the GA tracker:
https://github.com/rtfd/readthedocs.org/blob/499a74076f0debae09acc1d4c4eeeab03f8f557e/readthedocs/doc_builder/templates/doc_builder/include.js.tmpl#L7-L12

Also, i don't think we want to be loading this more than once.